### PR TITLE
HTML for 3rd task 

### PR DESCRIPTION
**Pull request by alimalik3007**

### CODE REVIEW

File: .idea/watcherTasks.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (junaidrt) [inline .idea/watcherTasks.xml]: @alimalik3007  Why have we pushed temporary files on git ?

File: 3rd-task/index.html [hunk 2]
```diff
@@ -0,0 +1,374 @@
...
+</head>
+<body>
+<div id="wrapper">
+    <header class="nav-header flex">
+    <nav id="nav" class="container flex">
+    <ul class="flex">
+        <li><a href="index.html">Home</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Features</a></li>
+        <li><a href="#">Services</a></li>
```
Comment (junaidrt) [inline 3rd-task/index.html]: Adding `#` in `href` will scroll our page to the top. Can we please fix this one ?

File: 3rd-task/index.html [hunk 3]
```diff
@@ -0,0 +1,374 @@
...
+        <li>
+            <h3>HTML Website</h3>
+            <span class="line"></span>
+            <p>
+                Free One Year Domain <br/>
+                Free Installation <br/>
+                Free 1GB Linux Hosting <br/>
+                Unlimited Revision <br/>
+                Five Alternative Designs <br/>
+                Lifetime Support <br/>
```
Comment (junaidrt) [inline 3rd-task/index.html]: Why are we using `br` tags for line breaks in paragraph ? Is this a good approach ? What is the best practice to do in this case ?

File: css/style.scss [hunk 4]
```diff
@@ -0,0 +1,359 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600;700;800;900&display=swap');
+$light-gry:#e9e9e9;
+$cgreen-color:#10c9c3;
+$white-color:#fff;
+$dark-brown:#191919;
+$gry:#838383;
+$section-gap:100px 0;
+$container:1170px;
+$center:0 auto;
```
Comment (junaidrt) [inline css/style.scss]: Please move all the variables in another scss file and import that file in our global file ?

**Review by junaidrt — CHANGES_REQUESTED** [review]

File: 3rd-task/index.html (hunk 2 above)
Comment (alimalik3007) [reply 3rd-task/index.html]: we have design just main home page design

File: 3rd-task/index.html (hunk 3 above)
Comment (alimalik3007) [reply 3rd-task/index.html]: I have no use span and p for other lines am just fusing br and move on text next line

File: 3rd-task/style.css [hunk 5]
```diff
@@ -0,0 +1,396 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600;700;800;900&display=swap");
```
Comment (junaidrt) [inline 3rd-task/style.css]: Do we have to push `.css` as well ?

File: css/style.scss [hunk 6]
```diff
@@ -0,0 +1,359 @@
...
+}
+.project-text p {
+    margin: 6px 0 0 0;
+    font-size: 12px;
+    color: #fff;
+    text-transform: uppercase;
+}
+// #gallery{
+//     display: flex;
+// }
```
Comment (junaidrt) [inline css/style.scss]: Please remove commented out code.

File: css/style.scss [hunk 7]
```diff
@@ -0,0 +1,359 @@
...
+#gallery ul{
```
Comment (junaidrt) [inline css/style.scss]: I'm unable to fine any id containing `gallery`.

**Review by junaidrt — CHANGES_REQUESTED** [review]

File: css/style.scss (hunk 6 above)
Comment (alimalik3007) [reply css/style.scss]: ok i will

Comment (alimalik3007) [reply css/style.scss]: it's second task SCSS?

File: 3rd-task/index.html (hunk 3 above)
Comment (junaidrt) [reply 3rd-task/index.html]: I dont think so it is a good approach. Please google it and find an other better way to do this.

File: .idea/watcherTasks.xml (hunk 1 above)
Comment (alimalik3007) [reply .idea/watcherTasks.xml]: am just push my coding folder on git so am sorry am not seen these temporary files

File: css/style.scss (hunk 4 above)
Comment (alimalik3007) [reply css/style.scss]: ok

File: css/style.scss (hunk 7 above)
Comment (alimalik3007) [reply css/style.scss]: i have not seen this id in css and Sass file in 3rd task file

File: 3rd-task/style.css (hunk 5 above)
Comment (alimalik3007) [reply 3rd-task/style.css]: ok i will

File: .idea/watcherTasks.xml (hunk 1 above)
Comment (junaidrt) [reply .idea/watcherTasks.xml]: Please remove these files from this PR.

File: css/style.scss (hunk 7 above)
Comment (junaidrt) [reply css/style.scss]: Then I think we should remove the redundant css from the file.

**Review by junaidrt — CHANGES_REQUESTED** [review]

File: .idea/watcherTasks.xml (hunk 1 above)
Comment (alimalik3007) [reply .idea/watcherTasks.xml]: ok

File: css/style.scss (hunk 7 above)
Comment (alimalik3007) [reply css/style.scss]: it's second task css

Comment (junaidrt) [reply css/style.scss]: Is our second task related to the 3rd one ?

File: 3rd-task/index.html (hunk 3 above)
Comment (alimalik3007) [reply 3rd-task/index.html]: ok sure i will check

File: 3rd-task/index.html [hunk 8]
```diff
@@ -0,0 +1,375 @@
...
+            <h3>HTML Website</h3>
+            <span class="line"></span>
+            <p>
+                <p>Free One Year Domain <p/>
+                 <p>Free Installation</p>
+                <p>Free 1GB Linux Hosting <p/>
+               <p> Unlimited Revision <p/>
+               <p> Five Alternative Designs <p/>
+               <p> Lifetime Support <p/>
+                <p>Moneyback Guaranteed<p/>
```
Comment (junaidrt) [inline 3rd-task/index.html]: Please use a single `p` tag instead of multiple for each line.

File: 3rd-task/style.css [hunk 9]
```diff
@@ -0,0 +1,400 @@
...
```
Comment (junaidrt) [inline 3rd-task/style.css]: Do we really have to push `.css` file on git while we are writing `.scss`

File: 3rd-task/style.css.map [hunk 10]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (junaidrt) [inline 3rd-task/style.css.map]: We don't have to push `.css.map` to git ?

File: .idea/watcherTasks.xml (hunk 1 above)
Comment (junaidrt) [inline .idea/watcherTasks.xml]: Please remove this temporary file.

File: css/style.scss (hunk 4 above)
Comment (junaidrt) [inline css/style.scss]: Why these variables are still defined here ? 
We have defined a separate file for variables.

**Review by junaidrt — CHANGES_REQUESTED** [review]

**Pull request merged**